rep/main.py

```diff
@@ -16,6 +16,8 @@
 
 voteObjectsDao = VoteObjectDao()
 
+time.sleep(10)
```

Comment:
This shouldn't get checked in, unless you want to throw a flag in here that causes this to happen only when we're doing development, which you could do with something like
```python
if os.getEnv("ENV", "prod") == "local":
  time.sleep(10)
```

Then you'd run the program like 
```
ENV=local repp --process
```

Reply:
Ok yeah I'll change it to that. I can't see myself spending time creating a better debugging arrangement, and I'll definitely forget to remove the time.sleep() again if I have to add it and remove it every time I merge something